PVM_PVP/Any/ModernSlayer.py

```
@@ -0,0 +1,493 @@
+            if any(keyword in weapon_name for keyword in weapon_keywords):
+                return weapon
+            
+            # Additional check: look at item properties for weapon-related properties
+            try:
+                props = API.ItemNameAndProps(weapon.Serial, wait=True)
+                if props:
+                    props_lower = props.lower()
+                    # Check for weapon-related properties
+                    if any(prop in props_lower for prop in ["slayer", "damage", "durability", "weapon", "swing", "thrust", "shoot"]):
+                        return weapon
+            except:
```

Ruff (0.13.1)

Do not use bare `except`

(E722)